scalafix-sbt/src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -54,7 +60,29 @@ object ScalafixPlugin extends AutoPlugin {
     cliWrapperMainClass := "scalafix.cli.Cli$",
     scalafixEnabled := true,
     scalafixVerbose := false,
-    sbtfix := Def.inputTaskDyn {
+    sbtfix := sbtfixImpl().evaluated,
+    sbtfixTest := sbtfixImpl(true).evaluated,
+    aggregate.in(sbtfix) := false,
+    aggregate.in(sbtfixTest) := false,
+    scalafixSourceroot := baseDirectory.in(ThisBuild).value,
+    scalafixVersion := Versions.version,
+    scalafixSemanticdbVersion := Versions.scalameta,
+    scalafixScalaVersion := Versions.scala212,
+    cliWrapperClasspath := {
+      val jars = ScalafixJarFetcher.fetchJars(
+        "ch.epfl.scala",
+        s"scalafix-cli_${scalafixScalaVersion.value}",
+        scalafixVersion.value
+      )
+      if (jars.isEmpty) {
+        throw new MessageOnlyException("Unable to download scalafix-cli jars!")
+      }
+      jars
+    }
+  )
+
+  private def sbtfixImpl(runAsTest: Boolean = false) = {
```

Comment:
Instead of `runAsTest: Boolean`, can  we propagate `extraOptions: Seq[String]`? That would make it easier to extend for other flags in the future.

Reply:
Sure! Would you like the same implementation for `scalafixTest` ?

Reply:
Yes, that would be great

Reply:
Fixed